### ISSUE TITLE
Error java.lang.LinkageError when start sofa-ark-samples using this plugin

### ISSUE THREAD

**Issue by ggndnn** [ggndnn/sofa-ark-idea-plugin#3]
Got a  java.lang.LinkageError when start sofa-ark-samples (master branch whose ark dependency was upgraded to 1.0.0).

```
Exception in thread "main" com.alipay.sofa.ark.exception.ArkRuntimeException: com.alipay.sofa.ark.exception.ArkRuntimeException: java.lang.reflect.InvocationTargetException
	at com.alipay.sofa.ark.container.pipeline.StandardPipeline.process(StandardPipeline.java:81)
	at com.alipay.sofa.ark.container.ArkContainer.start(ArkContainer.java:143)
	at com.github.ggndnn.sofa.ark.execution.model.SofaArkLauncher.main(SofaArkLauncher.java:31)
Caused by: com.alipay.sofa.ark.exception.ArkRuntimeException: java.lang.reflect.InvocationTargetException
	at com.alipay.sofa.ark.container.service.biz.DefaultBizDeployer.deploy(DefaultBizDeployer.java:56)
	at com.alipay.sofa.ark.container.service.biz.BizDeployServiceImpl.deploy(BizDeployServiceImpl.java:55)
	at com.alipay.sofa.ark.container.pipeline.DeployBizStage.process(DeployBizStage.java:47)
	at com.alipay.sofa.ark.container.pipeline.StandardPipeline.process(StandardPipeline.java:75)
	... 2 more
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.alipay.sofa.ark.bootstrap.MainMethodRunner.run(MainMethodRunner.java:48)
	at com.alipay.sofa.ark.container.model.BizModel.start(BizModel.java:215)
	at com.alipay.sofa.ark.container.service.biz.DefaultBizDeployer.deploy(DefaultBizDeployer.java:52)
	... 5 more
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'defaultRuntimeShutdownAware' defined in class path resource [com/alipay/sofa/boot/autoconfigure/runtime/SofaRuntimeAutoConfiguration.class]: Initialization of bean failed; nested exception is java.lang.LinkageError: loader constraint violation: when resolving interface method "com.alipay.sofa.runtime.spi.component.SofaRuntimeManager.registerShutdownAware(Lcom/alipay/sofa/runtime/spi/spring/RuntimeShutdownAware;)V" the class loader (instance of com/alipay/sofa/ark/container/service/classloader/BizClassLoader) of the current class, com/alipay/sofa/runtime/spring/RuntimeShutdownAwarePostProcessor, and the class loader (instance of com/alipay/sofa/ark/container/service/classloader/PluginClassLoader) for the method's defining class, com/alipay/sofa/runtime/spi/component/SofaRuntimeManager, have different Class objects for the type com/alipay/sofa/runtime/spi/spring/RuntimeShutdownAware used in the signature
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:601)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:515)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:320)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:318)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:845)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:877)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:549)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:742)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:389)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311)
	at sample.Sample3Application.main(Sample3Application.java:28)
	... 12 more
Caused by: java.lang.LinkageError: loader constraint violation: when resolving interface method "com.alipay.sofa.runtime.spi.component.SofaRuntimeManager.registerShutdownAware(Lcom/alipay/sofa/runtime/spi/spring/RuntimeShutdownAware;)V" the class loader (instance of com/alipay/sofa/ark/container/service/classloader/BizClassLoader) of the current class, com/alipay/sofa/runtime/spring/RuntimeShutdownAwarePostProcessor, and the class loader (instance of com/alipay/sofa/ark/container/service/classloader/PluginClassLoader) for the method's defining class, com/alipay/sofa/runtime/spi/component/SofaRuntimeManager, have different Class objects for the type com/alipay/sofa/runtime/spi/spring/RuntimeShutdownAware used in the signature
	at com.alipay.sofa.runtime.spring.RuntimeShutdownAwarePostProcessor.postProcessAfterInitialization(RuntimeShutdownAwarePostProcessor.java:57)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsAfterInitialization(AbstractAutowireCapableBeanFactory.java:429)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1782)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:593)
	... 24 more
```

**Comment by ggndnn** [comment]
在sofa-boot提交了[issue](https://github.com/sofastack/sofa-boot/issues/478).

**Comment by ggndnn** [comment]
This issue will be fixed in sofa-boot v3.2.1.
